samples/rbac.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cosi-controller-sa
+  # replace with non-default namespace name
+  namespace: default
+
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cosi-controller-manager-role
+rules:
+  - apiGroups: ["objectstorage.k8s.io"]
+    resources: ["bucketrequests", "bucketaccessrequests"]
+    verbs: ["get", "list", "watch", "create", "delete"]
+  - apiGroups: ["objectstorage.k8s.io"]
+    resources: ["buckets", "bucketaccess"]
+    verbs: ["get", "list", "watch", "update", "create", "delete"]
+  - apiGroups: ["objectstorage.k8s.io"]
+    resources: ["buckets","bucketrequests", "bucketclass","bucketaccess","bucketaccessrequests", "bucketaccessclass"]
+    verbs: ["get", "list", "watch"]
```

Comment:
Does the central controller "watch" BCs and BACs?

Reply:
Fixed